cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,58 @@
-/// <reference types='cypress' />
-/// <reference types='../support' />
+/// <reference types="cypress" />
+/// <reference types="../support" />
 
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
+
+const settingsPage = new SettingsPageObject();
+const signInPage = new SignInPageObject();
 describe('Settings page', () => {
+  let user;
   before(() => {
-
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
   });
 
   beforeEach(() => {
-
+    cy.task('db:clear');
+    cy.visit('/');
+    cy.register();
+    signInPage.visit();
+    signInPage.typeEmail('riot@qa.team');
+    signInPage.typePassword('12345Qwert!');
+    signInPage.clickSignInBtn();
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(2000);
```

Comment:
Using `cy.wait(2000)` is generally discouraged as it can lead to flaky tests. It's better to wait for specific elements or conditions using commands like `cy.get()` or `cy.contains()` with `.should('be.visible')` to ensure the application is in the expected state before proceeding.